### PR TITLE
Show seats for schedule 

### PR DESCRIPTION
**Pull request by Frlundsten**
Display schedules and seats for a movie.

Move seats into schedule so that the data can go straight from clicked schedule to the displayed seat.
Add schedule prop to seats component to be able to show the seats from that schedule.
Map bookings to the correct schedule in schedules so that the clicked schedule always shows the correct amount of seats left.

### CODE REVIEW

File: frontend/src/components/Seats.vue [hunk 1]
```diff
@@ -1,32 +1,23 @@
 <script setup>
-import { ref, onMounted } from "vue";
+import { ref, onMounted, defineEmits } from "vue";
```
Comment (Mattan41) [inline frontend/src/components/Seats.vue:2]: `defineEmits` is a compiler macro and no longer needs to be imported =)

**Review by Mattan41 — APPROVED** [review]
🥇

**Comment by DevelopedByDanny** [comment]
Should we show something when a movie does not have a schedule?  eg. "Not showing" or something

**Review by DevelopedByDanny — APPROVED** [review]
Nice job!!

**Pull request merged**